README.md

```diff
@@ -183,19 +206,64 @@ The default level passed along with every log sent through this instance.
 
 The default MAC Address passed along with every log sent through this instance.
 
-### Log(line)
----
-#### Options
+##### MaxBufferLen
 
-##### Level
+* _**Optional**_
+* Type: `int`
+* Default: `5`
+* Values: `10`
+
+MaxBufferLen sets the number of logs that are buffered before data is flushed and shipped to LogDNA.
+
+##### Meta
+
+* _**Optional**_
+* Type: `Meta`
```

Comment:
I think this should be `struct`, no?

Reply:
it's a struct called Meta- and when declaring the variable you specify the type as Meta

Reply:
We refer to the variable type when we say `Type` in this way and `meta` has a `Struct` type.